core/src/content/gitignore_fs.ts

```diff
@@ -38,10 +38,29 @@ export class GitignoreFs extends FileSystem {
 }
 
 function isTextExtension(name: string) {
-  const overrides = ["go", "ts"];
-  if (overrides.includes(name.split(".").pop() || "")) {
-    return true;
+  // If theres no extension, err on the side
```

Comment:
Why not an allowlist?

Reply:
No reason not to other than I don't know the allows we need.

Reply:
Either way we miss some. This is probably "safer" that it'll error and fail, rather than silently do nothing.